Local Pods/GitHubAPI/GitHubAPI/V3RepositoryNotificationRequest.swift

```diff
@@ -13,7 +13,21 @@ public struct V3RepositoryNotificationRequest: V3Request {
     public var pathComponents: [String] {
         return ["repos", owner, repo, "notifications"]
     }
+    public var parameters: [String : Any]? {
+        return [
+            "all": all ? "true" : "false",
```

Comment:
Why not like this: ` "all": all.description` ?

Reply:
Because I copied it over and didn't know about this returning the exact strings. Nice, changed!